src/modules/launcher/Plugins/Microsoft.Plugin.Folder/ContextMenuLoader.cs

```diff
@@ -59,6 +60,39 @@ public List<ContextMenuResult> LoadContextMenus(Result selectedResult)
                         }
                     }
                 });
+
+                contextMenus.Add(new ContextMenuResult
+                {
+                    PluginName = Assembly.GetExecutingAssembly().GetName().Name,
+                    Title = _context.API.GetTranslation("Microsoft_plugin_folder_open_in_console"),
+                    Glyph = "\xE756",
+                    FontFamily = "Segoe MDL2 Assets",
+                    SubTitle = $"Open current {fileOrFolder} path in console",
```

Comment:
Should translated as Title above?

Reply:
@somil55 As discussed offline.  I'm okay with removing the subtitle field.  It's remnant of the wox implementation that leveraged the results list for context menu items.

Reply:
@martinchrzan I have updated this PR with the changes you requested. Can you please take a look at it?